benchflow-dsl/src/main/scala/cloud/benchflow/dsl/BenchFlowDSL.scala

```diff
@@ -159,4 +161,30 @@ object BenchFlowDSL {
 
   }
 
+  /**
+   *
```

Comment:
It is going to be fixed in next pull requests. See https://github.com/benchflow/benchflow/issues/382